p5js/ImageClassification/sketch.js

```diff
@@ -29,6 +28,6 @@ function imageReady() {
 // When we get the results
 function gotResult(results) {
   // The results are in an array ordered by probability.
-  select('#result').html(results[0].label);
+  select('#result').html(results[0].className);
```

Comment:
Will "className" be more confusing than "label"? Is it important to use the underlying terminology? (I mean these to be open questions.)

Reply:
I change this because I came across much more references to 'class' and 'class name' rather than 'label' in other examples.